Microsoft.Identity.Web/Client/TokenAcquisition.cs

```diff
@@ -392,7 +392,8 @@ private void AddAccountToCacheFromJwt(IEnumerable<string> scopes, JwtSecurityTok
                 IEnumerable<string> requestedScopes;
                 if (jwtToken != null)
                 {
-                    userAssertion = new UserAssertion(jwtToken.RawData, "urn:ietf:params:oauth:grant-type:jwt-bearer");
+                    string rawData = (jwtToken.InnerToken != null) ? jwtToken.InnerToken.RawData : jwtToken.RawData;
```

Comment:
LGTM, but this library really needs to have unit tests as the complexity is raising

Reply:
I agree. I was lazy ...